### PR TITLE
Proposal UI

### PR DESCRIPTION
**Pull request by Megha-Dev-19**
image
image

### CODE REVIEW

File: src/indexer/index.ts [hunk 1]
```diff
@@ -40,12 +40,12 @@ export const getApplicationGlobalState = async (
 		// get global state of application
 		const globalStateRes = await executeReq(lookupApplications(appId));
 		if (
-			globalStateRes?.allApps?.nodes.length &&
-			JSON.parse(globalStateRes.allApps.nodes[0].params).gs
+			globalStateRes?.allSigmaDaos?.nodes.length &&
```
Comment (rmeena840) [inline src/indexer/index.ts]: why??? We don't have sigma_daos table now. We are now directly using the app table now. Please setup sigmadao-indexer repo.

**Review by rmeena840 — CHANGES_REQUESTED** [review]
You are trying to revert the changes that I did to make sigmadao compatible with sigmadao-indexer.
+ Please setup sigmadao-indexer. Clear your DB first.
+ I have already made sigmadao compatible with sigmadao-indexer. You only need to run it and you should be able to see the dao and proposal as before.

File: src/indexer/index.ts (hunk 1 above)
Comment (Megha-Dev-19) [reply src/indexer/index.ts]: Yes I have to update it
I forgot to add a TODO in the description.
I needed to test it and the indexer PR wasn’t merged till then thus reverted back 
Will update

**Comment by Megha-Dev-19** [comment]
> You are trying to revert the changes that I did to make sigmadao compatible with sigmadao-indexer.
> 
> * Please setup sigmadao-indexer. Clear your DB first.
> * I have already made sigmadao compatible with sigmadao-indexer. You only need to run it and you should be able to see the dao and proposal as before.

okay

File: src/pages/AllDao.vue [hunk 2]
```diff
@@ -68,9 +68,7 @@
 			<h4 class="margin_left_sm">Fetching Data</h4>
 		</div>
 		<div v-else>
-			<a-empty description="No Sigma DAOs Exists">
-				<a-button type="primary" @click="handleCreateDAO">Create</a-button>
-			</a-empty>
+			<a-empty description="No Sigma DAOs Exists" />
```
Comment (rmeena840) [inline src/pages/AllDao.vue]: Let's move it to constant file

**Review by rmeena840 — APPROVED** [review]
lgtm. check comments

**Pull request merged**